src/wallet/wallet2.cpp

```diff
@@ -12137,6 +12137,18 @@ uint64_t wallet2::get_daemon_blockchain_target_height(string &err)
   return target_height;
 }
 
+bool wallet2::is_daemon_busy_syncing()
+{
+  if (m_offline || !tools::is_local_address(get_daemon_address()))
```

Comment:
Why the local check ?

Reply:
Moved the check into `doRefresh(..)`. We care about local daemons only.